### PR TITLE
Changed css to fit smaller screens

### PR DESCRIPTION
**Pull request by jennygaoo**

### CODE REVIEW

File: postal-injection/src/main/webapp/data_display.html [hunk 1]
```diff
@@ -106,14 +106,12 @@ <h4><b>Enter Your Parameters</b></h4>
     <!-- Slideshow Start -->
     <div id="captions_container">
       <div class="captions">
-        <!-- TODO(ksharpe): write caption. Feel free to change title -->
         <h3><b>Order Destination Heat Map</b></h3>
-        <p>Description here</p>
+        <p>Quickly visualize all destinations for all orders provided in the data.</p>
```
Comment (jennygaoo) [inline postal-injection/src/main/webapp/data_display.html:110]: should've put this in a different PR, since it's not directly related

Comment (andreahu12) [reply postal-injection/src/main/webapp/data_display.html:110]: it's ok sometimes we slide changes in ;)
